tests/test_dataset.py

```diff
@@ -0,0 +1,254 @@
+import os
+import posixpath
+
+import pytest
+
+import audeer
+import audiofile
+import audb
+import audformat
+import audbcards
+import audplot
+
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+
+
+CACHE = './cache'
```

Comment:
This might lead to the problems I experienced as the user might have different content in that folder.
One idea would be to delete that folder at the beginning of the tests, but better would be to provide the cache folder by a fixture, e.g. you could add to `conftest.py`
```python
@pytest.fixture
def cache(tmpdir, scope='function'):
    return audeer.mkdir(audeer.path(tmpdir, 'cache'))
```
and then use the fixture inside `test_dataset()` with `test_dataset(tmpdir, db, cache)`

Reply:
Yes you're right I had to delete the cache everytime before I run the tests. I've applied the change and added the fixture now.

Reply:
In a future PR let's handle the caching of the `Dataset` class itself in a better way but this PR as you said let's keep the current solution.

Reply:
Cool, thanks.